docs/modules/operation/pages/deep-dive/topology/enlinkd/introduction.adoc

```diff
@@ -57,6 +57,15 @@ CDP, LLDP, OSPF and ISIS collectors provide the required information about links
 Bridge Collectors are able to collect Bridge Forwarding Table and IpNetToMedia Table on devices and to provide effective links a further step is required.
 The Enlinkd-Bridge-Discovery process attempts to discover bridge domain links with the data coming from all collected Bridge Forwarding Tables.
 The Topology-Updaters correlate all collected topology data to generate a global topology layout of your network.
+For each supported protocol is created a collection group with an assigned priority.
+The collection group is scheduled according with the rescan interval defined for the related protocol.
+The run of collection group will send to a Priority Executor protocol specific Snmp Collection and storage.
```

Comment:
```suggestion
The collection group run sends protocol-specific SNMP collection and storage to a priority executor.
```
I might have rephrased this incorrectly, so please verify.

Reply:
Only one priority Executor.